src/main/scala/com/typesafe/sbt/packager/docker/Keys.scala

```diff
@@ -68,4 +68,8 @@ private[packager] trait DockerKeysEx extends DockerKeys {
   )
   val dockerLayerMappings =
     taskKey[Seq[LayeredMapping]]("List of layer, source file and destination in Docker image.")
+  val chownUser =
+    SettingKey[Option[String]]("chown-user", "Optional alternative user to use in chown in COPY command")
+  val chownGroup =
+    SettingKey[Option[String]]("chown-group", "Optional alternative group to use in chown in COPY command")
```

Comment:
```suggestion
  val dockerChownUser =
    SettingKey[Option[String]]("docker-chown-user", "Optional alternative user to use in chown in COPY command")
  val dockerChownGroup =
    SettingKey[Option[String]]("docker-chown-group", "Optional alternative group to use in chown in COPY command")
```

If the setting is specific to a packaging format it's prefixed with that format. In this case `docker<SettingKey>`. The configuration scoping is not necessary, so you can move it in the plugins definition to the other settings starting with docker.